src/scripts/exclusion_term_expansion.py

```diff
@@ -120,7 +120,12 @@ def sparql_jinja2_robot_query(
             f.write(instantiated_str)
         with open(command_save_path, 'w') as f:
             f.write(command_str)
-        subprocess.run(command_str.split())
+        result = subprocess.run(command_str.split(), capture_output=True, text=True)
```

Comment:
@matentzn Now capturing output of `robot` and raising error where it occurs. I should've done this before.

Reply:
excellent